react-components/src/components/CustomerOrders/customerOrders.js

```diff
@@ -0,0 +1,82 @@
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TablePagination from '@material-ui/core/TablePagination';
+import TableRow from '@material-ui/core/TableRow';
+
+const CustomerOrders = props => {
+    const [t] = useTranslation(['account']);
+    const { data: customerOrdersData } = useQuery(QUERY_CUSTOMER_ORDERS);
+    const [page, setPage] = React.useState(0);
+    const [rowsPerPage] = React.useState(10);
+
+    const handleChangePage = (event, newPage) => {
+        setPage(newPage);
+    };
+
+    if (customerOrdersData) {
+        return (
+            <Paper className={classes.root}>
+                <TableContainer component={Paper}>
+                    <Table className={classes.table} aria-label="simple table">
+                        <TableHead>
+                            <TableRow>
+                                <TableCell>{t('customer-order-number', 'Order #')}</TableCell>
+                                <TableCell align="right">{t('customer-order-created-at', 'Date')}</TableCell>
+                                <TableCell align="right">{t('customer-order-grand-total', 'Total')}</TableCell>
+                                <TableCell align="right">{t('customer-order-status', 'Status')}</TableCell>
+                            </TableRow>
+                        </TableHead>
+                        <TableBody>
+                            {customerOrdersData.customerOrders.items
+                                .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage).map((row) => (
+                                    <TableRow key={row.order_number}>
+                                        <TableCell component="th" scope="row">
+                                            {row.order_number}
+                                        </TableCell>
+                                        <TableCell align="right">{row.created_at.substring(0, 10)}</TableCell>
+                                        <TableCell align="right">{row.grand_total}</TableCell>
+                                        <TableCell align="right">{row.status}</TableCell>
+                                    </TableRow>
+                                ))}
+                        </TableBody>
+                    </Table>
+                </TableContainer>
+                <TablePagination
+                    rowsPerPageOptions={[]}
+                    component="div"
+                    count={customerOrdersData.customerOrders.items.length}
+                    rowsPerPage={rowsPerPage}
+                    page={page}
+                    onChangePage={handleChangePage}
+                    labelRowsPerPage={""}
+                />
+            </Paper>
+        );
+    } else {
+        return "";
```

Comment:
When there are no orders, we should put some message like: "You have placed no orders yet".